tmcprototype/SdpSubarrayLeafNode/SdpSubarrayLeafNode/SdpSubarrayLeafNode.py

```diff
@@ -87,7 +87,7 @@ def commandCallback(self, event):
 
 
     SdpSubarrayNodeFQDN = device_property(
-        dtype='str', default_value="mid_sdp/elt/subarray_1",
+        dtype='str', default_value="tango://dada:10000/mid_sdp/elt/subarray_1",
```

Comment:
Remove hard coded tango host value.

Reply:
Code is already updated with default value